dt-assets/functions/enqueue-scripts.php

```diff
@@ -197,9 +197,18 @@ function dt_site_scripts() {
                         "delete" => strtolower( __( "Delete", "disciple_tools" ) ),
                         "translate" => __( "Translate with Google Translate", "disciple_tools" ),
                         "hide_translation" => __( "Hide Translation", "disciple_tools" ),
+                        "reaction_title_1" => _x( '{{user}} reacted with {{emojiName}} emoji', 'Bob reacted with heart emoji (dont translate words in brackets)', 'disciple_tools' ),
```

Comment:
@squigglybob can you change the {{key}} to use the %1$s,  %2$s etc
this will be more standard with the rest of the translation strings.
example: `'showing_x_items_matching' => _x( 'Showing %1$s items matching %2$s', 'Showing 30 items matching bob', 'disciple_tools' ),`